.env.example

```diff
@@ -14,6 +14,8 @@ DATABASE_URL=postgres://postgres:postgresyour-super-secret-and-long-postgres-pas
 # This is where your API runs in development
 PORT=4000
 LOG_LEVEL=info
+LOG_FLARE_API_KEY=
+LOG_FLARE_SOURCE_TOKEN=
```

Comment:
I'm a fan of fake values in the example .env because they give a hint on how the token / api keys are formated (especially when you deal with a bunch of em). But feel free to ignore. Not critical.